app/screens/user_profile/user_profile.js

```diff
@@ -97,6 +102,34 @@ class UserProfile extends PureComponent {
         return null;
     };
 
+    buildTimezoneBlock = () => {
```

Comment:
this one too, should only be available if the server has the timezone feature available